doc/introduction/chemistry.rst

```diff
@@ -43,7 +43,10 @@ installation. This backend is non-differentiable and can be selected by setting
 ``method='pyscf'`` in :func:`~.molecular_hamiltonian`. Additionally, if the electronic Hamiltonian
 is built independently using `OpenFermion <https://github.com/quantumlib/OpenFermion>`_ tools, it
 can be readily converted to a PennyLane observable using the
-:func:`~.pennylane.import_operator` function.
+:func:`~.pennylane.import_operator` function. There is also capability to import wavefunctions (states) 
+that have been pre-computed by configuration interaction singles and doubles (CISD) and coupled cluster 
+singles and doubles (CCSD) methods via PySCF, for example to provide a better starting point to a quantum algorithm. 
```

Comment:
From a maintainability perspective, perhaps we don't want to mention the specific methods (e.g., if we add a few more, this becomes out of date).

Reply:
Makes sense, especially since we already have two new methods beyond this that we are aiming to add in the 0.33 release. Will make more generic.

Reply:
Done.